tests/core/test_utils.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 import unittest
 import os
+import pytest
```

Comment:
Ok, I got it.
Found a different solution with asserRaises()

Thank you for the tip :)